numba/tests/test_sets.py

```diff
@@ -171,36 +171,25 @@ def union_usecase(a, b):
     s = sa.union(set(b))
     return list(s)
 
-def set_return_usecase(a):
-    s = set(a)
-    return s
+def difference_usecase__2(a, b):
+    s = a.difference(b)
+    return list(s)
 
+def intersection_usecase__2(a, b):
+    s = a.intersection(b)
+    return list(s)
 
-def make_operator_usecase(op):
-    code = """if 1:
-    def operator_usecase(a, b):
-        s = set(a) %(op)s set(b)
-        return list(s)
-    """ % dict(op=op)
-    return compile_function('operator_usecase', code, globals())
+def symmetric_difference_usecase__2(a, b):
+    s = a.symmetric_difference(b)
+    return list(s)
 
-def make_inplace_operator_usecase(op):
-    code = """if 1:
-    def inplace_operator_usecase(a, b):
-        sa = set(a)
-        sb = set(b)
-        sc = sa
-        sc %(op)s sb
-        return list(sc), list(sa)
-    """ % dict(op=op)
-    return compile_function('inplace_operator_usecase', code, globals())
-
-def make_comparison_usecase(op):
-    code = """if 1:
-    def comparison_usecase(a, b):
-        return set(a) %(op)s set(b)
-    """ % dict(op=op)
-    return compile_function('comparison_usecase', code, globals())
+def union_usecase__2(a, b):
```

Comment:
These `__2` function names don't really note their intended use. Given they are used only in the unit tests in `TestSetsInvalid` perhaps just define them locally in those unit tests?